src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -47,13 +53,33 @@ public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
      */
     public AddressBook toModelType() throws IllegalValueException {
         AddressBook addressBook = new AddressBook();
+        //Wedding has to be added first, since person has a reference to wedding
```

Comment:
Person has a reference to the wedding too

How about we add Person first
and if the Person has a wedding, we add the wedding

Because every Wedding will have a Client assigned to it.

Reply:
that works too, i will try to change that